components/dashboard/filters/ComboSelectFilter.tsx

```diff
@@ -86,7 +87,8 @@ function ComboSelectFilter({
   value: any;
   isMulti?: boolean;
   selected?: string[];
-  options: { label: React.ReactNode; value: any }[];
+  options?: { label: React.ReactNode; value: any }[];
```

Comment:
ComboSelectFilter looks fairly abstract, we should move this to the root `filters` folder.